File: backend/java-spring/src/main/java/com/unosquare/worldcup/model/Team.java
package com.unosquare.worldcup.model;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Table(name = "teams")
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Team {

    @Id
    private String id;
    private String name;
    private String code;

    @Column(name = "group_name")
    private String groupName;
}

File: backend/python-flask/app/models/team.py
from app.db import db


class Team(db.Model):
    __tablename__ = 'teams'

    id = db.Column(db.String(50), primary_key=True)
    name = db.Column(db.String(100), nullable=False)
    code = db.Column(db.String(10), nullable=False)
    group_name = db.Column(db.String(5), nullable=False)

    def to_dict(self):
        return {
            'id': self.id,
            'name': self.name,
            'code': self.code,
            'group': self.group_name,
        }
